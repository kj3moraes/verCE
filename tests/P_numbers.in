12
1.0
0.1
99
1234567890.987654321