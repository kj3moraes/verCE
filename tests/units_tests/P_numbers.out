ID:x 
BECOMES:= 
NUM:12 
ID:x 
BECOMES:= 
NUM:1.0 
ID:x 
BECOMES:= 
NUM:0.1 
ID:x 
BECOMES:= 
NUM:99 
ID:x 
BECOMES:= 
NUM:1234567890.987654321 
//:eof://:eof 
Handling top level expression
Handling top level expression

Compilation failed!
ERROR: Unknown token when expecting an expression
